### DISCUSSION TITLE
0.5.2

### DISCUSSION THREAD

**Discussion by himself65** [toeverything/AFFiNE#1932] [Announcements]
## Editor - Database View

![image](https://user-images.githubusercontent.com/14026360/231899228-f8609e6c-1d8d-428b-a623-41f4a8c3d832.png)

## Single Page Sharing (AFFiNE Cloud)

You can easily share a single page with others with just one click.

![image](https://user-images.githubusercontent.com/14026360/231897758-12c4e9f9-a6c2-4a6b-8fce-2b88e68b1850.png)

## Desktop (preview)

We currently provide MacOS (Intel & ARM) and Window app.

![image](https://user-images.githubusercontent.com/14026360/231897543-5347db0f-01f6-4199-819f-c979356ea6a5.png)

## Noticeable Bug fixes

- fix(y-indexeddb): migration in firefox by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1904
- Optimize `@toeverything/y-indexeddb` #1869
- fix: export to html/markdown by @daochouwangu in https://github.com/toeverything/AFFiNE/pull/1892

## Detailed Changes
* ci: build macos on pull request by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1854
* docs: sign CLA by @lzlme in https://github.com/toeverything/AFFiNE/pull/1855
* ci: fix windows build artifacts path name error by @m1911star in https://github.com/toeverything/AFFiNE/pull/1856
* refactor: move non-affine hooks by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1857
* feat: connect pinboard and reference link by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1859
* feat: support page sharing by meta by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1858
* chore: bump version by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1863
* docs: thanks to all contributors, update the readme by @tzhangchi in https://github.com/toeverything/AFFiNE/pull/1864
* docs: remove the global center style by @tzhangchi in https://github.com/toeverything/AFFiNE/pull/1865
* fix: update i18n by @lzlme in https://github.com/toeverything/AFFiNE/pull/1860
* docs: make intro,badge,desc align center by @tzhangchi in https://github.com/toeverything/AFFiNE/pull/1866
* feat: add affine next theme by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1867
* feat: ensure subpage is appended in its parent after subpageLinked by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1872
* chore: bump version by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1873
* feat: support google cloud login in client by @m1911star in https://github.com/toeverything/AFFiNE/pull/1822
* fix(y-indexeddb): migration for once by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1868
* perf(y-indexeddb): improve boost and loading time by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1879
* feat: single page sharing support by @JimmFly in https://github.com/toeverything/AFFiNE/pull/1805
* refactor: workspace header by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1880
* docs(y-indexeddb): add README.md by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1886
* fix(component): affine loading by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1887
* feat: update favicon & manifest by @darkskygit in https://github.com/toeverything/AFFiNE/pull/1889
* chore: bump version by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1890
* fix: api server address in electron by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1888
* feat: refactor trash, page would delete from its parent's subpageIds after move to trash by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1871
* fix: update app icons by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1893
* chore: enable databse feature flag by @doodlewind in https://github.com/toeverything/AFFiNE/pull/1895
* fix: export to html/markdown by @daochouwangu in https://github.com/toeverything/AFFiNE/pull/1892
* fix: ws prefix url in electron by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1896
* fix: try fix electron build by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1899
* fix: cors header handling by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1900
* refactor: remove next/dynamic by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1901
* fix: use `startTransition` by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1903
* fix: improve UX of dropdown by @gyuannn in https://github.com/toeverything/AFFiNE/pull/1905
* fix: cannot update a component while rendering a different component by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1907
* build: support sentry replay by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1908
* build: support sourcemap in sentry by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1910
* fix: wrap React.lazy with Suspense by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1915
* fix(y-indexeddb): migration in firefox by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1904
* feat: add navigation path in quick search by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1920
* fix: pinboard operation menu disappear inexplicably when hover to menu from button by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1922
* fix: electron sourcemap issues by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1919
* fix: failed to load blobs in electron by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1927
* chore: add lint cache by @Simon-He95 in https://github.com/toeverything/AFFiNE/pull/1917
* feat: enable share menu by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1883

## New Contributors
* @lzlme made their first contribution in https://github.com/toeverything/AFFiNE/pull/1855
* @daochouwangu made their first contribution in https://github.com/toeverything/AFFiNE/pull/1892
* @gyuannn made their first contribution in https://github.com/toeverything/AFFiNE/pull/1905
* @Simon-He95 made their first contribution in https://github.com/toeverything/AFFiNE/pull/1917

**Full Changelog**: https://github.com/toeverything/AFFiNE/compare/v0.0.2-2023.04.08-alpha-kreta-ayer...v0.5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/toeverything/AFFiNE/releases/tag/v0.5.2'>0.5.2</a>.</em>

**Comment by Dtdhamu** [comment]
> ## Editor - Database View
> 
> ![image](https://user-images.githubusercontent.com/14026360/231899228-f8609e6c-1d8d-428b-a623-41f4a8c3d832.png)
> 
> ## Single Page Sharing (AFFiNE Cloud)
> 
> You can easily share a single page with others with just one click.
> 
> ![image](https://user-images.githubusercontent.com/14026360/231897758-12c4e9f9-a6c2-4a6b-8fce-2b88e68b1850.png)
> 
> ## Desktop (preview)
> 
> We currently provide MacOS (Intel & ARM) and Window app.
> 
> ![image](https://user-images.githubusercontent.com/14026360/231897543-5347db0f-01f6-4199-819f-c979356ea6a5.png)
> 
> ## Noticeable Bug fixes
> 
> - fix(y-indexeddb): migration in firefox by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1904
> - Optimize `@toeverything/y-indexeddb` #1869
> - fix: export to html/markdown by @daochouwangu in https://github.com/toeverything/AFFiNE/pull/1892
> 
> ## Detailed Changes
> * ci: build macos on pull request by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1854
> * docs: sign CLA by @lzlme in https://github.com/toeverything/AFFiNE/pull/1855
> * ci: fix windows build artifacts path name error by @m1911star in https://github.com/toeverything/AFFiNE/pull/1856
> * refactor: move non-affine hooks by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1857
> * feat: connect pinboard and reference link by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1859
> * feat: support page sharing by meta by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1858
> * chore: bump version by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1863
> * docs: thanks to all contributors, update the readme by @tzhangchi in https://github.com/toeverything/AFFiNE/pull/1864
> * docs: remove the global center style by @tzhangchi in https://github.com/toeverything/AFFiNE/pull/1865
> * fix: update i18n by @lzlme in https://github.com/toeverything/AFFiNE/pull/1860
> * docs: make intro,badge,desc align center by @tzhangchi in https://github.com/toeverything/AFFiNE/pull/1866
> * feat: add affine next theme by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1867
> * feat: ensure subpage is appended in its parent after subpageLinked by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1872
> * chore: bump version by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1873
> * feat: support google cloud login in client by @m1911star in https://github.com/toeverything/AFFiNE/pull/1822
> * fix(y-indexeddb): migration for once by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1868
> * perf(y-indexeddb): improve boost and loading time by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1879
> * feat: single page sharing support by @JimmFly in https://github.com/toeverything/AFFiNE/pull/1805
> * refactor: workspace header by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1880
> * docs(y-indexeddb): add README.md by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1886
> * fix(component): affine loading by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1887
> * feat: update favicon & manifest by @darkskygit in https://github.com/toeverything/AFFiNE/pull/1889
> * chore: bump version by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1890
> * fix: api server address in electron by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1888
> * feat: refactor trash, page would delete from its parent's subpageIds after move to trash by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1871
> * fix: update app icons by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1893
> * chore: enable databse feature flag by @doodlewind in https://github.com/toeverything/AFFiNE/pull/1895
> * fix: export to html/markdown by @daochouwangu in https://github.com/toeverything/AFFiNE/pull/1892
> * fix: ws prefix url in electron by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1896
> * fix: try fix electron build by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1899
> * fix: cors header handling by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1900
> * refactor: remove next/dynamic by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1901
> * fix: use `startTransition` by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1903
> * fix: improve UX of dropdown by @gyuannn in https://github.com/toeverything/AFFiNE/pull/1905
> * fix: cannot update a component while rendering a different component by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1907
> * build: support sentry replay by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1908
> * build: support sourcemap in sentry by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1910
> * fix: wrap React.lazy with Suspense by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1915
> * fix(y-indexeddb): migration in firefox by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1904
> * feat: add navigation path in quick search by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1920
> * fix: pinboard operation menu disappear inexplicably when hover to menu from button by @QiShaoXuan in https://github.com/toeverything/AFFiNE/pull/1922
> * fix: electron sourcemap issues by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1919
> * fix: failed to load blobs in electron by @pengx17 in https://github.com/toeverything/AFFiNE/pull/1927
> * chore: add lint cache by @Simon-He95 in https://github.com/toeverything/AFFiNE/pull/1917
> * feat: enable share menu by @Himself65 in https://github.com/toeverything/AFFiNE/pull/1883
> 
> ## New Contributors
> * @lzlme made their first contribution in https://github.com/toeverything/AFFiNE/pull/1855
> * @daochouwangu made their first contribution in https://github.com/toeverything/AFFiNE/pull/1892
> * @gyuannn made their first contribution in https://github.com/toeverything/AFFiNE/pull/1905
> * @Simon-He95 made their first contribution in https://github.com/toeverything/AFFiNE/pull/1917
> 
> **Full Changelog**: https://github.com/toeverything/AFFiNE/compare/v0.0.2-2023.04.08-alpha-kreta-ayer...v0.5.2
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/toeverything/AFFiNE/releases/tag/v0.5.2'>0.5.2</a>.</em>
